packages/editor-ui/src/modules/communityNodes.ts

```diff
@@ -63,6 +91,40 @@ const module: Module<ICommunityNodesState, IRootState> = {
 				context.commit('setLoading', false);
 			}
 		},
+		async uninstallPackage(context: ActionContext<ICommunityNodesState, IRootState>) {
+			context.commit('setLoading', true);
+			try {
+				await uninstallPackage(context.rootGetters.getRestApiContext, context.getters.getCurrentModalPackage.packageName);
+				context.commit('removePackageByName', context.getters.getCurrentModalPackage.packageName);
+			} catch(error) {
+				throw(error);
+			} finally {
+				context.commit('setLoading', false);
+			}
+		},
+		async updatePackage(context: ActionContext<ICommunityNodesState, IRootState>) {
+			context.commit('setLoading', true);
+			try {
+				const packageToUpdate = context.getters.getCurrentModalPackage;
```

Comment:
this should be a method param... that way it's reusable.

Reply:
👍